sqlglot/lineage.py

```diff
@@ -113,7 +113,7 @@ def to_node(
                 )
             else:
                 if table not in tables:
-                    tables[table] = Node(name=table, source=source, expression=source)
+                    tables[table] = Node(name=str(c), source=source, expression=source)
```

Comment:
`str(c)` is not very idiomatic in sqlglot, I'd rather use `c.sql()` here.

Reply:
got it, will switch to `c.sql()`